zipline/__main__.py

```diff
@@ -185,8 +185,8 @@ def _(*args, **kwargs):
 @click.option(
     '--trading-calendar',
     metavar='TRADING-CALENDAR',
-    default='NYSE',
-    help="The calendar you want to use e.g. LSE. NYSE is the default."
+    default='XNYS',
```

Comment:
I'd probably leave this be for now. This is part of the public-facing API of the CLI, and more users are going to know NYSE than XNYS.